ducktape/cluster/remoteaccount.py

```diff
@@ -371,6 +371,22 @@ def kill_process(self, process_grep_str, clean_shutdown=True, allow_fail=False):
         for pid in pids:
             self.signal(pid, sig, allow_fail=allow_fail)
 
+    def java_pids(self, match):
+        cmd = """jcmd | grep """ + match + """ | awk '{print $1}'"""
```

Comment:
Ah, I figured there was something I could do like that with awk.  Will try it...